crates/bevy_state/src/state/transitions.rs

```diff
@@ -38,6 +38,19 @@ pub struct OnTransition<S: States> {
 }
 
 /// Runs [state transitions](States).
+///
+/// By default, it will be triggered after `PreUpdate`, but
+/// you can manually trigger it at arbitrary times by creating an exclusive
+/// system to run the schedule.
+///
+/// ```rust
+/// use bevy_state::prelude::*;
+/// use bevy_ecs::prelude::*;
+///
+/// fn run_state_transitions(world: &mut World) {
```

Comment:
should we just have a exclusive system that does this? It's going to be a small annoyance when migrating to make your own.

Reply:
I don't feel strongly either way. It's definitely a bit easier, but I like reminding people that they can just run schedules.